lib/main.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require_relative 'game'
+
+# Main program execution
```

Comment:
I would recommend to remove unnecessary comments. The names of your components and functions should be self-describing.